packages/superset-ui-chart-controls/src/shared-controls/components/ColumnConfigControl/constants.tsx

```diff
@@ -33,12 +33,22 @@ export type SharedColumnConfigProp =
   | 'colorPositiveNegative'
   | 'columnWidth'
   | 'fractionDigits'
+  | 'emitTarget'
   | 'd3NumberFormat'
   | 'd3SmallNumberFormat'
   | 'd3TimeFormat'
   | 'horizontalAlign'
   | 'showCellBars';
 
+const emitTarget: ControlFormItemSpec<'Input'> = {
+  controlType: 'Input',
+  label: t('Emit Target'),
+  description: D3_FORMAT_DOCS,
+  defaultValue: 'this',
```

Comment:
I think the default value should be `''`, and in that case we should emit the original column name (as the logic states in `getEmitTarget`).